arche/src/com/troy_west/arche.clj

```diff
@@ -0,0 +1,123 @@
+(ns com.troy-west.arche
+  (:require [qbits.alia :as alia]
+            [qbits.alia.codec.default :as codec.default]
+            [qbits.alia.udt :as alia.udt])
+  (:import (com.datastax.driver.core Session
+                                     BoundStatement
+                                     SimpleStatement)))
+
+(definterface IBindable
+  (^com.datastax.driver.core.BoundStatement bind [statement-store]))
+
+(defn bindable-statement
+  [statement-key values codec]
+  (let [q            ""
```

Comment:
Need to pass a query string into the constructor for a SimpleStatement but we never plan to use it since that with come from the saved prepared statement so an empty string is passed through.